src/pages/Signup/Signup.js

```diff
@@ -0,0 +1,229 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router';
+import './Signup.scss';
+
+function Signup() {
+  const [name, setName] = useState('');
+  const [nickname, setNickname] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [passwordConfirm, setPasswordConfirm] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+  const [github, setGithub] = useState('');
+  const [description, setDescription] = useState('');
+  const [position, setPosition] = useState('');
```

Comment:
객체 형태인 하나의 state로 관리할 수도 있을 것 같습니다!

```suggestion
  const [inputData, setInputData] = useState({
    name: '',
    nickname: '',
    ~~~
  });
```

Reply:
소중한 리뷰 감사합니다!  이렇게 수정했어요! 감사합니다
`  const [inputData, setInputData] = useState({
    name: '',
    nickname: '',
    email: '',
    password: '',
    passwordConfirm: '',
    phoneNumber: '',
    github: '',
    profilePhoto: '',
    description: '',
    position: '',
  });`